extensions/gdx-box2d/gdx-box2d-gwt/src/com/badlogic/gdx/physics/box2d/gwt/emu/com/badlogic/gdx/physics/box2d/ContactImpulse.java

```diff
@@ -29,16 +29,14 @@
 	}
 
 	public float[] getNormalImpulses () {
-		for (int i = 0; i < impulse.count; i++) {
-			normalImpulses[i] = impulse.normalImpulses[i];
-		}
+		if (impulse.count >= 0)
+			System.arraycopy(impulse.normalImpulses, 0, normalImpulses, 0, impulse.count);
 		return normalImpulses;
 	}
 
 	public float[] getTangentImpulses () {
-		for (int i = 0; i < impulse.count; i++) {
-			tangentImpulses[i] = impulse.tangentImpulses[i];
-		}
+		if (impulse.count >= 0)
+			System.arraycopy(impulse.tangentImpulses, 0, tangentImpulses, 0, impulse.count);
```

Comment:
This seems good. Is the size check necessary? Can `impulse.count` ever be negative?